build/BUILD.libzmq

```diff
@@ -67,6 +67,12 @@ genrule(
 
 #if defined __APPLE__
   #define ZMQ_HAVE_OSX
+  #undef ZMQ_USE_EPOLL
```

Comment:
instead of using undefs, please move the `genrule` to the end of the file
and add a separate genrule for the config for osx (going off the one that `./configure` generated for your system)
then use `select` and `config_setting` to include the rule individually